test/tap/install-parse-error.js

```diff
@@ -39,7 +39,6 @@ test('failing to parse package.json should be error', function (t) {
     function (err, code, stdout, stderr) {
       if (err) throw err
       t.equal(code, 1, 'exit not ok')
-      t.similar(stderr, /npm ERR! Failed to parse json/)
```

Comment:
This assertion needs to stand. The error message it's checking for is from npm and so Node.js version doesn't matter.

If this assertion is failing it means that something is unexpected about the error message. The install failed but what error did it report?